### ISSUE TITLE
Unable to Launch Multiple Kuma Dataplanes on the Same Machine Due to Port Conflict

### ISSUE THREAD

**Issue by vovasoft** [kumahq/kuma#11824]
### What happened?

When using `kuma-dp run`, I am unable to start it twice locally because the second time it prompts: `kuma-dp.run error while running Kuma DP {"error": "listen tcp 127.0.0.1:9902: bind: address already in use"}`. There is no configuration for this port; does Kuma not support starting two dataplanes on a single machine? I just want to do a simple test. My two commands are as follows:

Command 1: `sudo kuma-dp run \
  --cp-address=https://localhost:5678/ \
  --dns-enabled=true \
  --dataplane-token-file=java-service-token \
  --dataplane="
  type: Dataplane
  mesh: default
  name: java-service
  networking: 
    address: 127.0.0.1
    inbound: 
      - port: 15000
        servicePort: 8080
        serviceAddress: 127.0.0.1
        tags: 
          kuma.io/service: java-service
          kuma.io/protocol: http
    outbound: 
      - port: 80  # Keep using port 80
        tags:
          kuma.io/service: golang-service
    admin:
      port: 9904"`.

Command 2: `sudo kuma-dp run \
  --cp-address=https://localhost:5678/ \
  --dns-enabled=true \
  --dataplane-token-file=golang-service-token \
  --dns-envoy-port=9905 \
  --dataplane="
  type: Dataplane
  mesh: default
  name: golang-service
  networking: 
    address: 127.0.0.1
    inbound: 
      - port: 15001
        servicePort: 9091  # Change service port to 9091
        serviceAddress: 127.0.0.1
        tags: 
          kuma.io/service: golang-service
          kuma.io/protocol: http
    outbound: 
      - port: 8081
        tags:
          kuma.io/service: java-service
      admin:
        port: 9905"`.

**Comment by vovasoft** [comment]
upload a question, version is kuma-2.8.4

**Comment by jijiechen** [comment]
Thanks for reporting.

Can you try the following steps and see if they help?
https://github.com/kumahq/kuma-website/issues/1997#issuecomment-2426879813

**Comment by vovasoft** [comment]
Modifying environment variables with export doesn't change the port 9902 used at startup, so it doesn't work. Today, I also tried simulating the demo on multiple machines and encountered various issues. I suggest you carefully follow the procedures in the README, especially this one: https://github.com/kumahq/kuma-counter-demo/blob/master/README.md. It feels like the project hasn't been maintained for a while. A single-machine demo is crucial for understanding Kuma and service mesh. When I tested with Consul, it took me no more than 10 minutes, but with Kuma, I couldn't complete a single demo in three days. Setting up a Kubernetes environment is not convenient. In general, I suggest you try following your own README procedures and run the demo yourselves.
